src/routes/safe/components/Apps/index.jsx

```diff
@@ -0,0 +1,210 @@
+// @flow
+import React, { useState, useEffect, useCallback } from 'react'
+import styled from 'styled-components'
+
+import ButtonLink from '../../../../components/layout/ButtonLink'
+import Loader from './ListContentLayout/Loader'
+import sendTransactions from './sendTransactions'
+import { Wrapper, Menu, Content, Footer, Nav } from './ListContentLayout/Layout'
+import List from './ListContentLayout/List'
+
+const StyledIframe = styled.iframe`
+  width: 100%;
+  height: 100%;
+  display: ${props => (props.shouldDisplay ? 'block' : 'none')};
+`
+const operations = {
+  SEND_TRANSACTIONS: 'sendTransactions',
+  GET_TRANSACTIONS: 'getTransactions',
+  ON_SAFE_INFO: 'onSafeInfo',
+  ON_TX_UPDATE: 'onTransactionUpdate',
+}
+
+type Props = {
+  web3: any,
+  safeAddress: String,
+  network: String,
+  createTransaction: any,
+}
+
+function Apps({ web3, safeAddress, network, createTransaction }: Props) {
+  const apps = [
+    {
+      id: 1,
+      name: 'Compound',
+      // url: 'http://localhost:3002',
+      url: 'https://gnosis-apps.netlify.com/',
+      iconUrl: 'https://compound.finance/images/compound-mark.svg',
+      description: '',
+      providedBy: { name: 'Gnosis', url: '' },
+    },
+    {
+      id: 2,
+      name: 'ENS Manager',
+      url: '',
+      iconUrl: 'https://app.ens.domains/static/media/ensIconLogo.4d995d23.svg',
+      description: '',
+      providedBy: { name: 'Gnosis', url: '' },
+    },
+    {
+      id: 3,
+      name: 'Uniswap',
+      url: '',
+      iconUrl:
+        'https://blobscdn.gitbook.com/v0/b/gitbook-28427.appspot.com/o/spaces%2F-LNun-MDdANv-PeRglM0%2Favatar.png?generation=1538584950851432&alt=media',
+      description: '',
+      providedBy: { name: 'Gnosis', url: '' },
+    },
+    {
+      id: 4,
+      name: 'Nexus Mutual',
+      url: '',
+      iconUrl:
+        'https://blobscdn.gitbook.com/v0/b/gitbook-28427.appspot.com/o/spaces%2F-LK136DM17k-0Gl82Q9B%2Favatar.png?generation=1534411701476772&alt=media',
+      description: '',
+      providedBy: {
+        name: 'Gnosis',
+        url: '',
+      },
+    },
+  ]
+
+  const [selectedApp, setSelectedApp] = useState(1)
+  const [appIsLoading, setAppIsLoading] = useState(true)
+  const [iframeEl, setframeEl] = useState(null)
+
+  const getSelectedApp = () => apps.find(e => e.id === selectedApp)
```

Comment:
useMemo?

Reply:
I think is not needed yet, there will be only a few values